releasing.md

```diff
@@ -0,0 +1,393 @@
+
+# Release Process
+
+Below you'll find the instructions to release a `knative.dev` repository.
+
+# Timetable
+
+We release the components of Knative every 6 weeks. All of these components must
+be moved to the latest "release" of all shared dependencies prior to each
+release.
+
+## First week of the rotation
+
+### Make sure you have the right permission
+
+Check to make sure you already are in the "Knative Release Leads" team in
+https://github.com/knative/community/blob/main/peribolos/knative.yaml and
+https://github.com/knative/community/blob/main/peribolos/knative-sandbox.yaml
+. If not, send a PR like
+[this one](https://github.com/knative/community/pull/209) to grant yourself some
+super power. Ensure to run `/hack/update-codegen.sh` to add yourself to the owner
+files as well, as part of the PR.
+
+### Create a release Slack channel
+
+Ask someone from the TOC to create a **release-`#`** Slack channel that will be
+used to help manage this release.
+
+## 14 days prior to the release
+
+### 1. Update the Knative releasability defaults
+
+Update the defaults in
+[knative-releasability.yaml](https://github.com/knative-sandbox/knobots/blob/main/workflow-templates/knative-releasability.yaml)
+to this release by creating a PR. [Example PR](https://github.com/knative-sandbox/knobots/pull/170).
+
+These changes will be propagated to the rest of Knative in the next round of
+workflow syncs. Check to ensure that it has indeed propagated and if not the 
+sync will have to manually triggered.
+
+### 2. Announce the imminent `pkg` cut
+
+Announce on **#general** that `pkg` will be cut in a week.
+
+## 7 days prior to the release
+
+### 1. Cut release branches of supporting repos
+
+The supporting repos are the base repos where we have common code and common
+scripts. For these repos (except **hack**), we follow the same release process as
+explained in [release a repository](#release-a-repository), but no prow job is
+executed, hence no git tag and Github release are produced.
+
+```
+NOTE: The supporting repos must be released in the correct order, so you need to check if the [dependencies should be updated](#aligning-the-dependencies) after each repo release. 
+```
```

Comment:
This doesn't render properly

https://github.com/knative/release/blob/91e1c5bd167b724e6248657d3d19c38310cca6ba/releasing.md#1-cut-release-branches-of-supporting-repos

Reply:
are you using Safari? I'll change it but wondering